server/app/views/applicant/EditTiClientView.java

```diff
@@ -57,30 +57,39 @@ public Content render(
       ApplicantPersonalInfo personalInfo,
       Http.Request request,
       Messages messages,
-      Long accountId,
-      Optional<Form<EditTiClientInfoForm>> editTiClientInfoForm,
-      boolean isSuccessfulSave) {
-
+      Optional<Long> accountIdToEdit,
+      Long applicantIdOfTi,
+      Optional<Form<TiClientInfoForm>> tiClientInfoForm) {
+    Optional<AccountModel> optionalAccountModel = Optional.empty();
+    String title = "Add new Client";
+    String pageHeader = "Add client";
+    String pageId = "add-client";
+    String successToast = messages.at(MessageKey.BANNER_NEW_CLIENT_CREATED.getKeyName());
+    if (accountIdToEdit.isPresent()) {
+      optionalAccountModel =
+          Optional.of(accountRepository.lookupAccount(accountIdToEdit.get()).get());
+      title = "Edit client information";
+      pageHeader = "Edit client";
+      pageId = "edit-client";
+      successToast = messages.at(MessageKey.BANNER_CLIENT_INFO_UPDATED.getKeyName());
+    }
+    boolean isSuccessfulSave = tiClientInfoForm.isPresent() && !tiClientInfoForm.get().hasErrors();
     HtmlBundle bundle =
         layout
             .getBundle(request)
-            .setTitle("Edit client info")
+            .setTitle(title)
             .addMainContent(
                 renderHeader(tiGroup.getName(), "py-12", "mb-0", "bg-gray-50"),
                 hr(),
-                renderSubHeader("Edit client").withId("edit-client").withClass("my-4"),
+                renderSubHeader(pageHeader).withId(pageId).withClass("my-4"),
                 renderBackLink(),
-                renderSuccessAlert(
-                    isSuccessfulSave,
-                    messages.at(MessageKey.BANNER_CLIENT_INFO_UPDATED.getKeyName())),
+                renderSuccessAlert(isSuccessfulSave, successToast),
```

Comment:
suggestion (blocking):  The success alert styles for adding a new client are different than for editing.  Please use `ViewUtils.makeAlert()` rather than `ViewUtils.makeAlertSlim()` for the new client alert.  There's also more text, which you'll find in our `messages.conf` file.  
![image](https://github.com/civiform/civiform/assets/134556723/0117a573-90c5-42a8-9d41-e2c60a757cfc)